packages/web/src/dev/Racer.tsx

```diff
@@ -20,49 +20,29 @@ export default React.memo(function Racer({
   const lineAnimationKeyframes = [
     {
       from: {
-        transform: [{ scaleX: 1 }],
+        transform: [{ translateX: -relativePoints }],
```

Comment:
Easiest thing to do here would be to animate width from `0 `to `X`, however animating width isn't as smooth as animating transforms. next best would be to set width to x and scale from 0 to 1 however if we just scale the line the coin is just floating all awkwardly, and if we scale the div they are both in the coin goes from skinny af to the oval we know and love. 

prior solution was to animate the line width and coin position as separate animations. which worked but meant they could get out of sync or just mess up as connor demoed. 

therefore set width as desired and then just translateX the entire thing to the left to start. (hiding anything that overflows) 

🎉